airbyte-integrations/base/build.gradle

```diff
@@ -0,0 +1,18 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+plugins {
+    id 'com.bmuschko.docker-remote-api' version '6.6.1'
+}
+
+def image = 'airbyte/base:dev'
+
+task imageName {
+    doLast {
+        println "IMAGE $image"
+    }
+}
+
+task buildImage(type: DockerBuildImage) {
+    inputDir = projectDir
+    images.add(image)
+    outputs.upToDateWhen { false }
```

Comment:
why false here? if nothing changes shouldn't we reuse?

Reply:
I need to figure out how to make it respect changes to files up the docker dependency tree. Until then, re-running is the only way to make it 100% safe (and the docker steps cache anyways).